jlm-hls/include/jlm-hls/cmdline.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 Magnus Sjalander <work@sjalander.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#ifndef JLM_JLMHLS_CMDLINE_HPP
+#define JLM_JLMHLS_CMDLINE_HPP
+
+#include <jlm/util/file.hpp>
+#include <jlm/util/Statistics.hpp>
+
+#include <string>
+#include <vector>
+
+namespace jlm {
+
+class optimization;
+
+enum class outputformat {fir, dot};
```

Comment:
Could we rename fir to firrtl? It is more descriptive.

Reply:
Removed